client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-large-preview.tsx

```diff
@@ -135,10 +137,61 @@ const PatternLargePreview = ( {
 		);
 	};
 
+	const deviceComputerViewportScale = () =>
+		frameRef.current!.clientWidth / DEVICE_COMPUTER_VIEWPORT_WIDTH;
+
+	const setLargePreviewStyleForViewport = () => {
+		if ( NAVIGATOR_PATHS.SECTION_PATTERNS === navigator.location.path ) {
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg)

I think we don't need this condition as it's already in `useEffect`!